base/show.jl

```diff
@@ -335,13 +335,13 @@ show_linenumber(io::IO, line, file) = print(io," # ",file,", line ",line,':')
 # show a block, e g if/for/etc
 function show_block(io::IO, head, args::Vector, body, indent::Int)
     print(io, head, ' ')
-    show_list(io, args, ", ", indent)
+    show_list(io, args, ", ", indent, 0)
```

Comment:
I think this should stay at the default precedence level.  There aren't many cases where this could happen (particularly not sensible ones), but here's a failing test case:

```jl
julia> @test_repr """let @a(b), c=2
                        (b,c)
                     end"""
```

(Macro `@a` could be defined to return a syntax like `esc(:(b = 1))`).